services/auth-basic/README.md

```diff
@@ -0,0 +1,25 @@
+# Auth-Basic Service
+
+The `auth-basic` service is responsible for validating authentication of incoming requests. To do so it will forward the call to an `auth manager`
+(see `Auth Managers` section). Almost every request to the server will involve the `auth-basic` service as access tokens need to be verified.
+
+## Auth Managers
+
+Since the `auth-basic` service does not do any validation itself, it needs to be configured with an auth manager. One can use the `AUTH_BASIC_AUTH_PROVIDER` envvar to configure this.
```

Comment:
```suggestion
Since the `auth-basic` service does not do any validation itself, it needs to be configured with an authentication manager. One can use the `AUTH_BASIC_AUTH_PROVIDER` environment variable to configure this.
```
Just for completeness, checking the above env, I see that the description offers `ldap` but no other values. We should add a text / rewrite about possible values - even there is maybe currently only one present.